browser/src/react-redux/App/User/updateUser.action.js

```diff
@@ -0,0 +1,14 @@
+export const updateUser = user => ({
+  type: 'UPDATE_USER',
```

Comment:
You have an `UPDATE_USER` action, but nothing in the reducer to handle this. Let's build a case out in the reducer to update the user.